db/seeds.rb

```diff
@@ -244,7 +244,8 @@
   customize_experience: "Sloan here! 👋 Did you know that that you can customize your DEV experience? Try changing <a href='settings/ux'>your font and theme</a> and find the best style for you!",
   start_discussion: "Sloan here! 👋 I noticed that you haven't <a href='https://dev.to/t/discuss'>started a discussion</a> yet. Starting a discussion is easy to do; just click on 'Write a Post' in the sidebar of the tag page to get started!",
   ask_question: "Sloan here! 👋 I noticed that you haven't <a href='https://dev.to/t/explainlikeimfive'>asked a question</a> yet. Asking a question is easy to do; just click on 'Write a Post' in the sidebar of the tag page to get started!",
-  discuss_and_ask: "Sloan here! 👋 I noticed that you haven't <a href='https://dev.to/t/explainlikeimfive'>asked a question</a> or <a href='https://dev.to/t/discuss'>started a discussion</a> yet. It's easy to do both of these; just click on 'Write a Post' in the sidebar of the tag page to get started!"
+  discuss_and_ask: "Sloan here! 👋 I noticed that you haven't <a href='https://dev.to/t/explainlikeimfive'>asked a question</a> or <a href='https://dev.to/t/discuss'>started a discussion</a> yet. It's easy to do both of these; just click on 'Write a Post' in the sidebar of the tag page to get started!",
+  download_app: "Sloan here, with one last tip! 👋  Have you downloaded the DEV mobile app yet? Consider <a href=’https://dev.to/downloads>downloading</a> it so you can access all of your favorite DEV content on the go!"
```

Comment:
```suggestion
  download_app: "Sloan here, with one last tip! 👋  Have you downloaded the DEV mobile app yet? Consider <a href='https://dev.to/downloads>downloading'</a> it so you can access all of your favorite DEV content on the go!"
```
You'll need to close the url here, otherwise this will break when someone tries to seed. Also, note the character difference between `’` and `'`, I don't think Ruby will interpret what you have currently as a single quotation mark.

Reply:
I **just** noticed this and pushed up a change moment ago. Good catch! 😄